### DISCUSSION TITLE
Verified

### DISCUSSION THREAD

**Discussion by jstjohn24** [tweepy/tweepy#1554] [Questions] [answered]
Hello! Is there a way to filter out verified users in a stream?

**Comment by Harmon758** [comment, marked as answer]
[Tweepy v4.6.0](https://github.com/tweepy/tweepy/releases/tag/v4.6.0) has added support for Twitter API v2's filtered stream, which [supports an `is:verified` operator](https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/integrate/build-a-rule).

Just searching for `test.com` should work fine for picking up Tweets with links like `test.com/apple` or `apple.test.com`.
https://developer.twitter.com/en/docs/twitter-api/v1/tweets/filter-realtime/guides/basic-stream-parameters
